### PR TITLE
local-install.sh script for development purposes

### PR DESCRIPTION
**Pull request by greghuels**
Adding a script to the repo since others have found it useful for local development

### CODE REVIEW

File: scripts/local-install.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+if [ ! -d "$2" ]; then
+  echo "$2 is not a directory"
+  exit 1
+fi
+
+rm -rf ~/.yarn/berry/cache/\@eppo-*
+
+### Create local package via "npm pack"
+pushd "$1" > /dev/null
+rm -rf package pack.out pack.err && npm pack 2> pack.err > pack.out
```
Comment (aarsilv) [inline scripts/local-install.sh:37]: after using this script for a bit, would love for these intermediate files to be created in a temporary directory or something

Comment (greghuels) [reply scripts/local-install.sh:37]: I can make that happen!

Comment (greghuels) [reply scripts/local-install.sh]: updated!

File: scripts/local-install.sh [hunk 2]
```diff
@@ -1,17 +1,27 @@
...
+##  Install js-client-sdk-common changes into js-client-sdk:
+##   ./scripts/local-install.sh . ../js-client-sdk
+##
+##  Install js-client-sdk changes into a test app
+##   ./scripts/local-install ../js-client-sdk ../test-apps/my-react-app
+##
+##  Install js-client-sdk-common changes into a test app
+##   ./scripts/local-install.sh . ../js-client-sdk && ./scripts/local-install ../js-client-sdk ../test-apps/my-react-app
 
-## (requires jq (`brew install jq`))
```
Comment (greghuels) [inline scripts/local-install.sh]: I figured it's better to have this as a programmatic check than just a comment

File: scripts/local-install.sh [hunk 3]
```diff
@@ -1,17 +1,27 @@
...
+if ! command -v jq &> /dev/null
+then
+    echo "jq must be installed before using this script"
+    exit 1
+fi
 
 if [[ "$1" = "" || "$2" = "" ]]; then
-  echo 'USAGE: ./local-install.sh <package-to-install-path> <consuming-app-path>'
-  ehco ''
```
Comment (greghuels) [inline scripts/local-install.sh]: this was a typo, and we don't need spaces here anyway

File: scripts/local-install.sh [hunk 4]
```diff
@@ -30,15 +40,15 @@ if [ ! -d "$2" ]; then
...
 
-rm -rf ~/.yarn/berry/cache/\@eppo-*
-
 ### Create local package via "npm pack"
 pushd "$1" > /dev/null
-rm -rf package pack.out pack.err && npm pack 2> pack.err > pack.out
-COMPRESSED_PACKAGE="$(tail -1 pack.out)"
+mkdir -p /tmp/packages
+PACKAGE_DIR=/tmp/packages/"$(basename "$(pwd)")"
+rm -rf "${PACKAGE_DIR}" /tmp/pack.out && npm pack > /tmp/pack.out
```
Comment (greghuels) [inline scripts/local-install.sh:47]: piping errors to `pack.err` wasn't necessary and it suppressed errors when `npm pack` failed

File: scripts/local-install.sh [hunk 5]
```diff
@@ -30,15 +39,15 @@ if [ ! -d "$2" ]; then
   exit 1
 fi
 
-rm -rf ~/.yarn/berry/cache/\@eppo-*
```
Comment (greghuels) [inline scripts/local-install.sh]: This wasn't necessary

File: scripts/local-install.sh [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+if [ ! -f "$2/package.json" ]; then
+  echo "$2 is not an npm package"
+  exit 1
+fi
+
+### Create local package via "npm pack"
+pushd "$1" > /dev/null
+mkdir -p /tmp/packages
+PACKAGE_DIR=/tmp/packages/"$(basename "$(pwd)")"
```
Comment (aarsilv) [inline scripts/local-install.sh:50]: 🔥

**Review by aarsilv — APPROVED** [review]
Love this tool been using it already! 💪

**Pull request merged**